src/Symfony/Component/Finder/Tests/Comparator/ComparatorTest.php

```diff
@@ -43,9 +57,10 @@ public function testGetSetTarget()
      */
     public function testTest($operator, $target, $match, $noMatch)
```

Comment:
The data provider for this test does not provide full coverage (it tests only one operator). Would it make sense to add other cases ?

Reply:
The tests for this class are indeed pretty sparse. I think the class is mainly tested by testing its subclasses.

Reply:
#42164